api.js

```diff
@@ -1,10 +1,16 @@
 import axios from 'axios';
+import store from './store/store';
 
 const baseurl = 'http://localhost:5005/api';
 
-const config = {
-   'headers': {
-      'Content-Type': 'application/json'
+const getConfig = (auth = true) => {
+   var token = auth ? store.getState().auth.token : null;
```

Comment:
Is there a reason why you're using var instead of const?

Reply:
I have no reason 😂 I'll change it to const!